blockchain/flowkit.go

```diff
@@ -410,17 +410,14 @@ func (fk *flowKit) getAccountStorage(address flow.Address) (string, error) {
 	args := []string{fmt.Sprintf(`{"type":"Address","value":"0x%s"}`, address.Hex())}
 	val, _, err := fk.executeScript(StorageIteration, args)
 	if err != nil {
-		return "", err
+		return "{}", err
```

Comment:
usually when error is present you return a null value for the type, so "", why is this changed to return this?

Reply:
Changed back